sql-statements/sql-statement-show-character-set.md

```diff
@@ -26,16 +26,17 @@ SHOW CHARACTER SET;
 ```
 
 ```
-+---------+---------------+-------------------+--------+
-| Charset | Description   | Default collation | Maxlen |
-+---------+---------------+-------------------+--------+
-| utf8    | UTF-8 Unicode | utf8_bin          |      3 |
-| utf8mb4 | UTF-8 Unicode | utf8mb4_bin       |      4 |
-| ascii   | US ASCII      | ascii_bin         |      1 |
-| latin1  | Latin1        | latin1_bin        |      1 |
-| binary  | binary        | binary            |      1 |
-+---------+---------------+-------------------+--------+
-5 rows in set (0.00 sec)
++---------+-------------------------------------+-------------------+--------+
+| Charset | Description                         | Default collation | Maxlen |
++---------+-------------------------------------+-------------------+--------+
+| ascii   | US ASCII                            | ascii_bin         |      1 |
+| binary  | binary                              | binary            |      1 |
+| gbk     | Chinese Internal Code Specification | gbk_chinese_ci    |      2 |
```

Comment:
TiDB has supported GB18030 (master only now, not released in any version yet) https://github.com/pingcap/tidb/pull/55792 . Similar with gbk, its default collation is `gb18030_chinese_ci` when by default but `gb18030_bin` if new collation is not enabled. FWD @CbcWestwolf